### ISSUE TITLE
Support for --all-namespaces

### ISSUE THREAD

**Issue by xunholy** [instrumenta/conftest-action#3]
https://github.com/instrumenta/conftest-action/blob/ae9212d5c0c26e21eedf359e2889d0d5d39dabd4/action.yml#L29

I've like to be able to specify any number of the supported flags available in the conftest CLI can we have an input that allows me to specify my own flags? Or can we get support for `--all-namespaces` in the interim?

**Comment by garethr** [comment]
Adding an `args` property would definitely feel like a good idea for arbitrary flags. I've done this with other Actions and it works well.

**Comment by xunholy** [comment]
Agreed, this would then decouple the Github actions needing to maintain new inputs as the upstream CLI added new functionality. I also noticed here we're pinning the image to use `latest` we should also have the ability to version pin the Github action ideally, it needs to be a more conscious effort in case there could be introduced breaking changes or some fundamental functionality changes, EG. Flags, and other general upstream changes.

**Comment by garethr** [comment]
Alas you can't (or at least couldn't) pass a variable from the Action to the image. If you've seen a nice pattern for that I'd definitely be interested.

**Comment by xunholy** [comment]
@garethr correct the way it has been implemented does make it difficult.

I helped do something similar for the `ansible-lint-action` - https://github.com/ansible/ansible-lint-action

It wasn't exactly how I'd like but they now use `override-deps` as a way to cherrypick the version they want to install and use in their dockerfile by running custom logic in their entrypoint script.

The alternative is to explicitly use the image:tag with the version then release the action with tags so you can reference the action like following:

```
- name: Conftest
  uses: instrumenta/conftest-action@v0.18.2
```

I think either approach is fine, I find the latter requires more maintenance on this repo to ensure the new releases are maintained and tagged so the first option being able to parse the version puts that effort into the users' hands.
